thoth/storages/graph/postgres.py

```diff
@@ -3706,6 +3709,7 @@ def get_kebechet_github_installations_active_managers(self, slug: str) -> list:
                 session.query(KebechetGithubAppInstallations)
                 .filter(KebechetGithubAppInstallations.is_active.is_(True))
                 .filter(KebechetGithubAppInstallations.slug == slug)
+                .filter(KebechetGithubAppInstallations.runtime_environment_name == runtime_environment_name)
```

Comment:
This should be optional if runtime_environemnt is optional.

Reply:
Runtime environment is a nullable column, so this is a valid query that should always return a single row. I can remove the default value to make this more clear but it will require changes to packages using this function.